Makefile.kokkos

```diff
@@ -1378,19 +1378,72 @@ endif
 KOKKOS_EXTRA_LIBS := ${KOKKOS_LIBS}
 KOKKOS_LIBS := -lkokkos ${KOKKOS_LIBS}
 
+# Generating the <desul/atomics/Config.hpp> header
+DESUL_INTERNAL_CONFIG_TMP=Desul_Config.tmp
+ifeq ($(KOKKOS_INTERNAL_DISABLE_DESUL_ATOMICS), 0)
+  DESUL_CONFIG_HEADER=desul/atomics/Config.hpp
+else
+  DESUL_CONFIG_HEADER=NothingToSeeHereMoveAlong
+endif
```

Comment:
Any reason not to guard the whole following code block so that you only generate anything if (the bundled) `desul` is used?

Reply:
Did not want to indent the block and we are about to drop support for disabling Desul atomics

Reply:
I didn't mean to ask about an external `desul` here but only about using non-desul atomics.